ftva_lab_data/models.py

```diff
@@ -71,3 +74,15 @@ class SheetImport(models.Model):
 
     def __str__(self):
         return f"id: {self.id} --- file: {self.file_name} --- title: {self.title}"
+
+
+class ItemStatus(models.Model):
+    """Represents the status of an item in the SheetImport model."""
+
+    status = models.CharField(max_length=50, unique=True)
+
+    def __str__(self):
+        return self.status
+
+    class Meta:
+        verbose_name_plural = "Item statuses"
```

Comment:
Add `ordering = ["status"]` (instead of in the form, mentioned above)
https://docs.djangoproject.com/en/5.2/ref/models/options/#ordering